src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -17,21 +28,140 @@ const bgForType = (type: CardType) => {
 const Match = ({ match, label, bg }: { match: number; label: string; bg: string }) => {
   return (
     <div
-      className={`${bg} flex flex-row shrink-0 sm:flex-col rounded-[16px] sm:rounded-[40px] sm:min-h-[80px] w-[132px] sm:w-[80px] h-[32px] text-white justify-center text-center items-center`}
+      className={`${bg} flex flex-row shrink-0 sm:flex-col rounded-lg sm:rounded-[40px] sm:min-h-[80px] w-[132px] sm:w-[80px] h-[32px] text-white justify-center text-center items-center`}
     >
       <span className="mr-3 sm:mr-0 font-semibold text-[22px] sm:text-[24px]">{`${match}%`}</span>
-      <span className="flex justify-center text-[12px] font-bold">{label}</span>
+      <span className="flex justify-center text-[12px]/[16px] font-arial font-bold">{label}</span>
     </div>
   );
 };
-export const OpportunityCard = ({ name, description, matchValue, matchLabel, type }: OpportunityCardProps) => {
+
+const BottomBox = ({ title, children }: { title?: string; children: React.ReactNode }) => (
+  <div className="font-arial border border-inactive-gray py-2 px-3 -mr-[1px] -mb-[1px] text-[12px]/[20px] flex flex-row items-center gap-2">
+    <span className="flex items-center mr-1">{title}</span>
+    <span className="flex items-center">{children}</span>
+  </div>
+);
+
+const OutlookDots = ({ outlook, ariaDescription }: { outlook: number; ariaDescription: string }) => (
+  <div className="flex flex-row" aria-description={ariaDescription}>
```

Comment:
`aria-description` should be `aria-label`.

`aria-description` is not read in this case as the div element is not interactive.